sklearn/decomposition/pca.py

```diff
@@ -478,6 +512,9 @@ def score(self, X, y=None):
         return np.mean(self.score_samples(X))
 
 
+@deprecated("it will be removed in 0.19. Use PCA(svd_solver='randomized') "
```

Comment:
Use "RandomizedPCA will ..." rather than "it will ..." for a clearer deprecation message.